great_expectations/data_context/cloud_constants.py

```diff
@@ -1,3 +1,30 @@
+from enum import Enum
+
 from typing_extensions import Final
 
 CLOUD_DEFAULT_BASE_URL: Final[str] = "https://api.greatexpectations.io/"
+
+
+class GECloudEnvironmentVariable(str, Enum):
```

Comment:
`GE` vs `Ge` in these two `Enum`s? I prefer the uppercase version, but I know that isn't the convention across our codebase.